### DISCUSSION TITLE
Coverage map with/without beamforming

### DISCUSSION THREAD

**Discussion by Advaith2212** [NVlabs/sionna#246] [Q&A]
Hello, 

I wanted to see the variation of `coverage map` with and without the effect of beamforming in `Sionna RT`. 
    1. For a given scene, the transmitter and receiver were initially placed.
    2. Initially, the coverage map is computed and using `SVD`, the precoding vec has been found out. 
    3. The computed `precoding_vec` is passed as an argument to the `cm` function to see the effect of beamforming in coverage map. 

In both the instances, they appear to be similar. Can you help me with this.

**Comment by jhoydis** [comment]
Hi,

First of all, as soon as you have a transmitter with multiple antennas, precoding is assumed for the computation of the coverage map (as it would not be defined otherwise). Please check the [API documentation](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map). 

Second, it is probably not very useful to compute the precoding vector based on the SVD of a particular channel realization in this case. One would rather pick a precoding vector that points toward a specific direction. 

Third, could you share a code snippet that reproduces your results?
